webapp/src/webusb.tsx

```diff
@@ -42,22 +42,46 @@ export async function webUsbPairThemedDialogAsync(pairAsync: () => Promise<boole
 
     if (!await showConnectDeviceDialogAsync(confirmAsync))
         return notPairedResult();
-    if (!await showPickWebUSBDeviceDialogAsync(confirmAsync, implicitlyCalled))
-        return notPairedResult();
 
-    core.showLoading("pair", lf("Select your {0} and press \"Connect\".", boardName))
+    let connected = pxt.packetio.isConnected();
+
+    if (!connected && pxt.packetio.isConnecting()) {
+        const start = Date.now();
+        const TRY_FOR_MS = 2500;
+        core.showLoading("attempting-reconnect", lf("Attempting to reconnect automatically to your {0}", boardName));
```

Comment:
Simplify the message?

```suggestion
        core.showLoading("attempting-reconnect", lf("Reconnecting to your {0}", boardName));
```